### DISCUSSION TITLE
Thoughts on Hardware

### DISCUSSION THREAD

**Discussion by rkwesk** [ltsp/ltsp#183] [Q&A]
Back in August 2019 I wrote this based on Alkis’s posts concerning ltsp5.

https://wiki.debian.org/LTSP/Ltsp%20Hardware%20Requirements

Does this seem valid for ltsp19?

Richard

**Comment by alkisg** [comment]
Hi Richard,

I think it would be best if such LTSP documentation was maintained in the [community wiki](https://github.com/ltsp/community/wiki).
The LTSP code should make sure that distro-specific actions would be minimal; thus even distro-specific things should go in the LTSP community wiki.
So I'd recommend keeping the LTSP5 Debian documentation there, but putting all LTSP19+ documentation in the LTSP community wiki. Personally, in the future, I'll be helping as much as I can with the community wiki, but not with distro-specific wikies.

That said, some quick notes about the page you linked:

- switch vs hub: I don't think anyone is using hubs anymore, so I'd completely delete that paragraph
- gigabit vs 100 mbps: the server<=>switch connection and cabling needs to be gigabit; the clients<=>switch connection may be 100mbps.
- flow control: if even one client is 100 mbps, then the server must be directly connected to the switch, to avoid the flow control issue. Otherwise, if you're using chained switches, do an epoptes lan benchmark to make sure that the server can indeed send gigabit traffic to the clients.
- server hardware: since we're not supporting thin clients, and fat clients only have minimal impact on the server, I'd state "any recent PC with 2+ GB RAM and gigabit networking should be fine as an LTSP server. Also an SSD disk for both /srv/ltsp and /home might make things a bit faster".
- client hardware: "as LTSP client requirements are the same as live CD or local installation requirements, consult your distribution for client requirements. 2+ GB RAM and 1000+ cpubenchmark score are generally recommended".
